File: src/main/java/sisbase/mobbgone/MobBGone.java
package sisbase.mobbgone;

import org.bukkit.plugin.java.JavaPlugin;
import sisbase.mobbgone.handlers.SpawnHandler;

public final class MobBGone extends JavaPlugin {

    public static final Config cfg = new Config();
    @Override
    public void onEnable() {
        this.saveDefaultConfig();
        this.reloadConfig();
        cfg.initialize();
        getServer().getPluginManager().registerEvents(new SpawnHandler(),this);
    }

    @Override
    public void onDisable() {
    }
}

File: src/main/kotlin/sisbase/mobbgone/MobBGone.kt
package sisbase.mobbgone

import org.bukkit.plugin.java.JavaPlugin
import sisbase.mobbgone.handlers.SpawnHandler

class MobBGone : JavaPlugin() {
    override fun onEnable() {
        saveDefaultConfig()
        reloadConfig()
        cfg.initialize()
        server.pluginManager.registerEvents(SpawnHandler(), this)
    }

    override fun onDisable() {}

    companion object {
        val cfg = Config()
    }
}
